src/ru/skypro/Main.java

```diff
@@ -3,6 +3,88 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        // write your code here
+        task1();
+        task2();
+        task3();
+        task4();
+        task5();
+    }
+
+    public static void task1() {
+        byte clientOS = 1; //1 - Android, 0 - iOS
+        if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке ");
+        } else if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else System.out.println("Ошибка");
```

Comment:
если в других однострочных ветках используем скобки, то и в конечном else лучше соблюдать то же правило. Аналогично и для других мест
```suggestion
        } else {
            System.out.println("Ошибка");
        }
```